plan/physical_plans.go

```diff
@@ -156,9 +156,9 @@ type PhysicalIndexScan struct {
 	DoubleRead bool
 
 	// accessInAndEqCount is counter of all conditions in AccessCondition[accessEqualCount:accessInAndEqCount].
-	accessInAndEqCount int
+	AccessInAndEqCount int
 	// accessEqualCount is counter of all conditions in AccessCondition[:accessEqualCount].
```

Comment:
ditto

Reply:
Thank you for the sugguestion. I've changed the comments as follows:

`-  // accessEqualCount is counter of all conditions in AccessCondition[:accessEqualCount].`
`+  // AccessEqualCount is counter of all conditions in AccessCondition[:AccessEqualCount].`
`   AccessEqualCount int`